packages/alice_objectbox/lib/core/alice_core_objectbox.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:alice/model/alice_http_error.dart';
+import 'package:alice/model/alice_http_response.dart';
+import 'package:alice_objectbox/alice_store.dart';
+import 'package:alice_objectbox/model/cached_alice_http_call.dart';
+import 'package:alice_objectbox/model/cached_alice_http_error.dart';
+import 'package:alice_objectbox/objectbox.g.dart';
+import 'package:flutter/widgets.dart';
+
+class AliceCoreObjectBox extends AliceCore {
+  AliceCoreObjectBox(
+    super.navigatorKey, {
+    required AliceStore store,
+    required super.showNotification,
+    required super.showInspectorOnShake,
+    required super.notificationIcon,
+    required super.maxCallsCount,
+    super.directionality,
+    super.showShareButton,
+  }) : _store = store;
+
+  final AliceStore _store;
+
+  @override
+  Stream<List<AliceHttpCall>> get callsStream => _store.httpCalls
+      .query()
+      .order<int>(CachedAliceHttpCall_.createdTime, flags: Order.descending)
+      .watch(triggerImmediately: true)
+      .map((Query<CachedAliceHttpCall> query) => query.find())
+      .asBroadcastStream();
+
+  @override
+  List<AliceHttpCall> getCalls() => _store.httpCalls.getAll();
+
+  @override
+  Future<void> onCallsChanged([List<AliceHttpCall>? calls]) async {
+    if (calls?.isNotEmpty ?? false) {
+      notificationMessage = _getNotificationMessage();
+      if (notificationMessage != notificationMessageShown &&
+          !notificationProcessing) {
+        await _showLocalNotification();
+      }
+    }
+  }
+
+  String _getNotificationMessage() {
+    final int successCalls = (_store.httpCalls.query()
+          ..link(
+            CachedAliceHttpCall_.responseRel,
+            CachedAliceHttpResponse_.status
+                .greaterOrEqual(200)
+                .and(CachedAliceHttpResponse_.status.lessThan(300)),
+          ))
+        .build()
+        .count();
+
+    final int redirectCalls = (_store.httpCalls.query()
+          ..link(
+            CachedAliceHttpCall_.responseRel,
+            CachedAliceHttpResponse_.status
+                .greaterOrEqual(300)
+                .and(CachedAliceHttpResponse_.status.lessThan(400)),
+          ))
+        .build()
+        .count();
+
+    final int errorCalls = (_store.httpCalls.query()
+          ..link(
+            CachedAliceHttpCall_.responseRel,
+            CachedAliceHttpResponse_.status
+                .greaterOrEqual(400)
+                .and(CachedAliceHttpResponse_.status.lessThan(600)),
+          ))
+        .build()
+        .count();
+
+    final int loadingCalls = _store.httpCalls
+        .query(CachedAliceHttpCall_.loading.equals(true))
+        .build()
+        .count();
+
+    return <String>[
+      if (loadingCalls > 0) 'Loading: $loadingCalls',
+      if (successCalls > 0) 'Success: $successCalls',
+      if (redirectCalls > 0) 'Redirect: $redirectCalls',
+      if (errorCalls > 0) 'Error: $errorCalls',
+    ].join(' | ');
+  }
+
+  Future<void> _showLocalNotification() async {
```

Comment:
Can we keep it original `AliceCore`? I think we're repeating same code.

Reply:
Not easily doable as `AliceCore` uses a `BehaviorSubject` whereas `AliceCoreObjectBox` uses a call to ObjectBox, in other words
```dart
/// AliceCore
await flutterLocalNotificationsPlugin?.show(
  0,
  'Alice (total: ${callsSubject.value.length} requests)',
  message,
  notificationDetails,
  payload: '',
);

/// AliceCoreObjectBox
await flutterLocalNotificationsPlugin?.show(
  0,
  'Alice (total: ${_store.httpCalls.count()} requests)',
  message,
  notificationDetails,
  payload: '',
);
```